plugins/subscriber/index.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  tags: [
+    {
+      name: 'SUBSCRIBER',
+      permissions: {
+        public: true,
+        self: true,
```

Comment:
`self: true` allow any user to add it themselves! This should be `false` or not defined. See: https://coralproject.github.io/talk/docs/architecture/tags/

Reply:
This is likely to be added by a user plugin to sync with subscriber data, so having it `false` and not allowing any "user" to add it will ensure that the tag is managed.

Another approach could be that a user plugin could override/hook onto the tags to insert the subscriber tag based on user metadata instead.